packages/builder/src/pages/builder/app/[application]/settings/automations/index.svelte

```diff
@@ -38,8 +38,11 @@
   let loaded = false
   $: app = $appsStore.apps.find(app => $appStore.appId?.includes(app.appId))
   $: licensePlan = $auth.user?.license?.plan
-  $: page = $pageInfo.page
-  $: fetchLogs(automationId, status, page, timeRange)
+
+  // Reset the page every time that a filter gets updated
+  $: pageInfo.reset(), automationId, status, timeRange
```

Comment:
@aptkingston not sure if this the most "svelte-way" to achieve this. Could I get your thoughts?

Reply:
This is fine yea. We could really do with making a proper `AutomationLogFetch` or something at some stage because I really don't like this custom pagination implementation, but this reactive statement is totally fine in the meantime to achieve what we need.